core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java

```diff
@@ -78,6 +78,7 @@
     private final ConfigDocItemScanner configDocItemScanner = new ConfigDocItemScanner();
     private final Set<String> generatedAccessors = new ConcurrentHashMap<String, Boolean>().keySet(Boolean.TRUE);
     private final Set<String> generatedJavaDocs = new ConcurrentHashMap<String, Boolean>().keySet(Boolean.TRUE);
+    private final boolean generateDocs = !Boolean.parseBoolean(System.getProperty("skipDocs", "false"));
```

Comment:
Isn't the same as `Boolean.getBoolean("skipDocs")`?

Reply:
Thanks, fixed in 45cd863